odpf/stencil/v1/stencil.proto

```diff
@@ -89,15 +94,38 @@ message UploadDescriptorResponse {
 	string errors = 3;
 }
 
+message ListSnapshotsRequest {
+	string namespace = 1;
+	string name = 2;
+	string version = 3;
+	bool latest = 4;
+}
+
 message ListSnapshotsResponse {
 	repeated Snapshot snapshots = 1;
 }
 
-message ListSnapshotsRequest {
+message SearchRequest {
 	string namespace = 1;
 	string name = 2;
 	string version = 3;
 	bool latest = 4;
+	string query = 6;
+}
+
+message SearchResult {
+	message File {
+		string path = 1;
+		string package = 2;
+		repeated string messages = 3;
+		repeated string fields = 4;
+	}
+	Snapshot snapshot = 1;
+	repeated File files = 5;
```

Comment:
any reason we choose index 5 straight?

Reply:
Missed updating field number. we had couple of other fields in between before. Updated now.